maestro-client/build.gradle

```diff
@@ -71,6 +71,7 @@ dependencies {
     api(libs.jackson.module.kotlin)
     api(libs.jackson.dataformat.yaml)
     api(libs.jackson.dataformat.xml)
+    api("net.dongliu:apk-parser:2.6.10")
```

Comment:
@dmitry-zaitsev I was not able to reference my dependency here by `api(libs.apkParser)`. Am I missing something?

Reply:
I believe it should be `api(libs.apk.parser)`